src/datepicker/Datepicker.vue

```diff
@@ -164,6 +165,19 @@ export default defineComponent({
       required: false,
       default: false,
     },
+    /**
+     * Clears selected date
+     */
+    clearable: {
+      type: Boolean,
+      required: false,
+      default: false
+    }
+  },
+  methods: {
+    clearInput() {
+      this.input = null;
```

Comment:
This method actually does not clear the selection, it just clears the input text. Proper implementation should work just like `selectDay` currently does: emit the new `null` value and close the view if it is shown.